src/test/java/org/openrewrite/java/testing/junit5/UpdateBeforeAfterAnnotationsTest.java

```diff
@@ -148,12 +223,31 @@ void afterClass() {
                   }
               }
               """
+          ),
+          //language=kotlin
+          kotlin(
+            """
+              import org.junit.AfterClass
+
+              class Test {
+                  @AfterClass
+                  fun afterClass() {
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.api.AfterAll
+
+              class Test {
+                  @AfterAll
+                  fun afterClass() {
+                  }
+              }
+              """
           )
         );
     }
 
-    @Issue("https://github.com/openrewrite/rewrite/issues/150")
-    @Disabled
     @Test
```

Comment:
Ideally we retain this historical information, such that we have access to more context from the tests without having to annotate using the git history.
```suggestion
    @Issue("https://github.com/openrewrite/rewrite/issues/150")
    @Test
```